docs/Framework.md

```diff
@@ -230,7 +230,7 @@ Further explanations are provided in a dedicated [Managers](Managers.md) section
 
 #### NcManager
 
-NcManager is the base abstract manager control class for any manager control class in the control model. Manager control classes are singleton classes. Vendor-specific managers MUST be directly or indirectly derived from this control class.
+NcManager is the base abstract manager control class for any manager control class in the control model. Manager control classes are singleton classes. Non-standard managers created to expose vendor specific functionality MUST be directly or indirectly derived from this control class.
```

Comment:
Model rather than expose?
```suggestion
NcManager is the base abstract manager control class for any manager control class in the control model. Manager control classes are singleton classes. Non-standard managers created to model vendor specific functionality MUST be directly or indirectly derived from this control class.
```